test/com/facebook/buck/jvm/kotlin/testdata/kotlin_library_description/com/example/mixed/Example.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.good
```

Comment:
if the test succeeded even though these package names were incorrect I'm a little skeptical the test actually ran lol